src/brogue/GlobalsBase.c

```diff
@@ -0,0 +1,119 @@
+/*
```

Comment:
GlobalsBase.c contains global variables that are used to store game state and GlobalsBase.h contains constants that shouldn't change by variant (e.g. what white looks like).

Reply:
please add these as comments to the file